### PR TITLE
Better error for primitives on proxyLazy + fix StartupTimings

### PR DESCRIPTION
**Pull request by waresnew**
`findByPropsLazy` makes the properties of the lazy object also lazy, but that doesn't work for primitives bc `Reflect.get` can't be run on primitives

Before:

image

After:

image

### CODE REVIEW

File: src/utils/lazy.ts [hunk 1]
```diff
@@ -116,8 +116,12 @@ export function proxyLazy<T>(factory: () => T, attempts = 5, isChild = false): T
...
                     true
                 );
-
-            return Reflect.get(target[kGET](), p, receiver);
+            const obj = target[kGET]();
+            if (typeof obj === "object" || typeof obj === "function") {
+                return Reflect.get(obj, p, receiver);
+            } else {
+                return ()=>obj; // don't call Reflect.get if it's a primitive
+            }
```
Comment (Nuckyz) [inline src/utils/lazy.ts:124]:
```suggestion
            const lazyTarget = target[kGET]();
            if (typeof lazyTarget === "object" || typeof lazyTarget === "function") {
                return Reflect.get(lazyTarget, p, receiver);
            }

            const val = lazyTarget[p];
            return typeof val === "function" ? val.bind(lazyTarget) : val;
```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

**Comment by waresnew** [comment]
Thanks @Nuckyz for telling me how to actually fix startuptimings

File: src/utils/lazy.ts [hunk 2]
```diff
@@ -116,8 +116,11 @@ export function proxyLazy<T>(factory: () => T, attempts = 5, isChild = false): T
                     attempts,
                     true
                 );
-
-            return Reflect.get(target[kGET](), p, receiver);
+            const lazyTarget = target[kGET]();
+            if (typeof lazyTarget === "object" || typeof lazyTarget === "function") {
+                return Reflect.get(lazyTarget, p, receiver);
+            }
+            throw new Error("proxyInner called on a primitive value");
```
Comment (Nuckyz) [inline src/utils/lazy.ts:123]:
```suggestion
            throw new Error("proxyLazy called on a primitive value");
```

File: src/plugins/startupTimings/index.tsx [hunk 3]
```diff
@@ -17,9 +17,9 @@
 */
 
 import { Devs } from "@utils/constants";
-import { LazyComponent } from "@utils/react";
 import definePlugin from "@utils/types";
 
+import StartupTimingPage from "./StartupTimingPage";
```
Comment (Nuckyz) [inline src/plugins/startupTimings/index.tsx:22]:
```suggestion
import StartupTimingPage from "./StartupTimingPage";

```

**Review by Nuckyz — CHANGES_REQUESTED** [review]

File: src/utils/lazy.ts (hunk 2 above)
Comment (waresnew) [reply src/utils/lazy.ts:123]: 💀 mbbbbb

**Review by Nuckyz — APPROVED** [review]

**Pull request merged**